swagger.yml

```diff
@@ -177,6 +177,257 @@ paths:
             $ref: '#/definitions/sniCertificate'
         default:
           $ref: '#/responses/error'
+  /accounts/{account_id}/env:
+    get:
+      tags:
+        - environmentVariables
+      operationId: getEnvVars
+      parameters:
+        - name: account_id
+          description: Scope response to account_id
+          type: string
+          required: true
+          in: path
+        - name: context
+          description: Filter by deploy context
+          type: string
+          enum:
+            - all
+            - dev
+            - branch-deploy
+            - deploy-preview
+            - production
+          required: false
+          in: query
+        - name: scope
+          description: Filter by scope
+          type: string
+          enum:
+            - builds
+            - functions
+            - runtime
+            - post_processing
+          required: false
+          in: query
+        - name: site_id
+          description: 'If specified, only return environment variables set on this site'
+          type: string
+          required: false
+          in: query
+      responses:
+        '200':
+          description: OK
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/envVar'
+        default:
+          $ref: '#/responses/error'
+      description: >-
+        [Beta] Returns all environment variables for an account or site. An
+        account corresponds to a team in the Netlify UI.
+    post:
+      tags:
+        - environmentVariables
+      operationId: createEnvVars
+      description: >-
+        [Beta] Creates new environment variables. Granular scopes are available
+        on Pro plans and above.
+      parameters:
+        - in: body
+          name: env_vars
+          schema:
+            type: array
+            items:
+              type: object
+              properties:
+                key:
+                  type: string
+                  description: >-
+                    The existing or new name of the key, if you wish to rename it (case-sensitive)
+                scopes:
+                  description: >-
+                    The scopes that this environment variable is set to (Pro plans and
+                    above)
+                  type: array
+                  items:
+                    type: string
+                    enum:
+                      - builds
+                      - functions
+                      - runtime
+                      - post_processing
+                values:
+                  type: array
+                  items:
+                    $ref: '#/definitions/envVarValue'
+        - name: account_id
+          description: Scope response to account_id
```

Comment:
[Dust] I didn't think about this in the initial review. Should this be `Scope request` instead of `Scope response`? (for some or all of the commands, I'm not sure 🤔 )

Reply:
Hmm good Q, I think either would be correct. 6 to one, half-dozen to another :-)